### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,47 @@
...
 
-Cortex is an API compatible [Prometheus](https://prometheus.io)
-implementation, that natively supports multitenancy and scale-out clustering.
+Install the Cortex agent (an unmodified OSS Prometheus) on your Kubernetes cluster using the following command:
 
-Cortex is a [Weaveworks](https://weave.works) project that forms a major part
-of [Weave Cloud](https://cloud.weave.works). If you have any questions regarding
-our hosted Prometheus as a service, please
-contact [help@weave.works](mailto:help@weave.works).
+    $ kubectl -n kube-system apply -f 'https://cloud.weave.works/k8s/cortex.yaml?t=<token>'
```
Comment (rade) [inline README.md]: We should update the onboading page with this.

Comment (tomwilkie) [inline README.md]: @davkal ^^

**Comment by jml** [comment]
What's the motivation for this change?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,47 @@
-# Cortex: A multitenant, horizontally scalable Prometheus as a service
+<p align="center"><img src="imgs/logo.png" alt="Weave Cortex Logo"></p>
+
+# Opensource, horizontally scalable Prometheus as a service
```
Comment (jml) [inline README.md]: "Open source"

File: README.md [hunk 3]
```diff
@@ -1,36 +1,47 @@
...
-You will need v1.2.1 or later.
 
-When you've got Prometheus, you will need
-to
-[configure it to discover your services](https://prometheus.io/docs/operating/configuration/) and
-also configure it to send its data to Weave Cloud by adding the following
-top-level stanza to `prometheus.yml`:
+## About Weave Cortex
+
+To learn more about Weave Cortex consult the follow documents / talks:
```
Comment (jml) [inline README.md]: Please add a comma after "Weave Cortex", e.g. "…about Weave Cortex, consult…"

File: README.md [hunk 4]
```diff
@@ -1,36 +1,47 @@
...
+
+- [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
+- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/weaveworks/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
+- Or join us on [our Slack channel](https://slack.weave.works) and ask us directly!
+
+## Using Cortex on other platforms
+
+Cortex can be used to monitor virtually any application on any platform, as the local agent is a specially configure, unmodified OSS Prometheus binary.  You can [download Prometheus from its website](https://prometheus.io/download/).  You will need v1.2.1 or later.
```
Comment (jml) [inline README.md]: "configured"

File: README.md (hunk 1 above)
Comment (jml) [inline README.md]: Filed https://github.com/weaveworks/service-ui/issues/36

File: README.md (hunk 4 above)
Comment (jml) [inline README.md]: Also, https://github.com/weaveworks/service-conf/pull/329 updated *our* retrievers to be 1.3.1. There's no justification on the PR, and I've forgotten whether that change was essential to that PR (which purported to fix links) or incidental. 

Which is all a rather long way of saying, should this be 1.3.1, not 1.2.1?

**Comment by tomwilkie** [comment]
> What's the motivation for this change?

Just trying to improve the readme.

File: README.md (hunk 4 above)
Comment (tomwilkie) [inline README.md]: 1.2.1 is still fine.  We updated to 1.3.1 for the k8s service discovery, which allows us to scrape multiple ports per pod, which we needed to improve the consul monitoring / alerting.

**Pull request merged**